src/main/java/com/lorepo/icplayer/client/page/KeyboardNavigationController.java

```diff
@@ -60,6 +60,7 @@ public final class KeyboardNavigationController implements IKeyboardNavigationCo
 	private int parentScrollY = 0;
 	private int iframeOffsetTop = 0;
 	private int parentWindowHeight = 0;
+	private boolean isNVDAStatusWasChanged = false;
```

Comment:
Ta zmienna zmienia tylko raz wartość przy "public void handleNVDAAvability(boolean shouldUseNVDA)", gdzie od tamtego momentu jest zawsze true. Biorąc pod uwagę, że to ta zmienna jest wykorzystywana w setRoleApplication to nie da się jej wyłączyć bez utworzenia na nowo obiektu KeyboardNavigationController.java. Należy pamiętać, że w PlayerController jest on używany jako final "private final KeyboardNavigationController keyboardController = new KeyboardNavigationController();"

Nieoczekiwane zachowanie jakie zaobserwowałem na lekcji testowej.
1. Kliknięcie guzika Select powoduje, że od tego momentu włączanie TTS i jego wyłączanie nie dodaje atrybutu "aria-hidden"
2. Od kliknięcie guzika powoduje, że od tego momentu włączanie TTS i jego wyłączanie nie usuwa atrybutu "aria-hidden".
3. Nie ważne czy kliknie się ponownie guzik to nie można wyjść z drugiego trybu.